### PR TITLE
Fixed Typo

### PR DESCRIPTION
**Pull request by gopavasanth**
Fixed typo #4

### CODE REVIEW

**Comment by monsij** [comment]
@gopavasanth : Henceforth make a PR from a branch of your fork and not the master branch. Also try to follow the PR template

File: pro1/pro1/settings.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'catalog.apps.CatalogConfig',
+    'noticeboard',
```
[the remark this answers is not in the document]
Comment (monsij) [reply pro1/pro1/settings.py]: @gopavasanth  Please look into this

[the remark this answers is not in the document]
Comment (gopavasanth) [reply pro1/pro1/settings.py]: I guess we can also add apps in this way? Referance: https://stackoverflow.com/questions/35354239/include-application-in-django-settings-py

[the remark this answers is not in the document]
Comment (monsij) [reply pro1/pro1/settings.py]: Is it running locally?

[the remark this answers is not in the document]
Comment (gopavasanth) [reply pro1/pro1/settings.py]: Yes I guess :-)

**Comment by monsij** [comment]
Good job! Squash your commits into one with a suitable commit log. Then it will be merged!

**Pull request closed without merging**